drivers/serial/uart_stm32.c

```diff
@@ -1463,12 +1464,12 @@ static int uart_stm32_init(const struct device *dev)
 		ll_parity = LL_USART_PARITY_NONE;
 		ll_datawidth = LL_USART_DATAWIDTH_8B;
 	}
-
+	ll_stopbits = uart_stm32_cfg2ll_stopbits(config->stop_bits);
 	/* Set datawidth and parity, 1 start bit, 1 stop bit  */
```

Comment:
could the `uart_stm32_cfg2ll_stopbits()`  raise an error if configuring a non-supported stop-bit
For example DTS defines 0.5 but the soc/uart does not support